addons/repositories.tf

```diff
@@ -25,7 +25,10 @@ data "helm_repository" "weaveworks" {
 # Initialize and destroy helm / tiller
 resource "null_resource" "helm_init_client" {
   provisioner "local-exec" {
-    command = "helm init --wait --service-account ${kubernetes_cluster_role_binding.tiller_clusterrolebinding.subject[0].name}"
+    command = <<EOF
+    helm init --wait --service-account ${kubernetes_cluster_role_binding.tiller_clusterrolebinding.subject[0].name};
+    kubectl --namespace=kube-system patch deployment tiller-deploy --type=json --patch='[{"op": "add", "path": "/spec/template/spec/containers/0/command", "value": ["/tiller", "--listen=localhost:44134"]}]'
```

Comment:
I would prefer if the additional commands were customizable by variable, this is a good default to have, but we'll need a different setup for our cluster.

Reply:
Note sure what you mean. Line 29 is exactly as it was before, just not using a multiline command.
The new part in line 30 only has impact if you are using helm/tiller from on the cluster. If that is necessary then there is probably more to be done: https://engineering.bitnami.com/articles/helm-security.html

I'm not sure what _vairables_ there are to customise?

Reply:
Any more feedback on this?

Reply:
Sorry missed the previous comment. I would suggest that line 30 is converted to a terraform variable.